components/profile_popover/profile_localtime.tsx

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {DateTime, Duration} from 'luxon';
+
+import {FormattedMessage} from 'react-intl';
+
+import {UserTimezone} from '@mattermost/types/users';
+import Timestamp from 'components/timestamp';
+
+type ProfileTimezoneProps = {
+    profileUserTimezone?: UserTimezone;
+    currentUserTimezone: string | undefined;
+}
+
+const returnTimeDiff = (
+    currentUserTimezone: string | undefined,
+    profileUserTimezone: UserTimezone | undefined,
+) => {
+    if (!currentUserTimezone || !profileUserTimezone) {
+        return undefined;
+    }
+    const currentUserDate = DateTime.local().setZone(currentUserTimezone);
+    const profileUserDate = DateTime.local().setZone(profileUserTimezone?.manualTimezone || profileUserTimezone?.automaticTimezone);
```

Comment:
Same re: `getUserCurrentTimezone`

Reply:
This value actually comes from the props on the Profilepopover and I just pass it through. I can probably use the suggested function here if you'd like! Curious if it's already used up the chain though. That's why I didn't have a check manual vs automatic

Reply:
`currentUserTimezone` is indeed preselected upstream in profile_popover's index, but `profileUserTimezone` is the raw timezone config that still needs to be determined according to the auto/manual preference.

Reply:
Okay, that's what I thought! So, I added logic for the `profileUserTimezone` and will leave `currentUserTimezone` alone since it's already selected